### DISCUSSION TITLE
Scala 3.0

### DISCUSSION THREAD

**Discussion by Reeebuuk** [plewand/scala-cosmos#3] [Ideas]
Hi there :)

Great work on showing the Scala 3 capabilities. Would be great if this project and blog post is ported to released Scala 3.0 version since some things have changed meanwhile :/

I've tried but some 50 compile error are there hoho

**Comment by plewand** [comment]
Thanks. The code is updated to compile now, tested successfully with sbt and IntelliJ nightly build plugin.

**Reply by Reeebuuk** [reply]
Awesome, was so close but so far :)
